### ISSUE TITLE
Datadog: customizable additional tags for metrics

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#160]
### Is your feature request related to a problem? Please describe.
We have deployments in the cloud (autoscaling or failover models, CSTv1)
We need some additional data (in form of tags) to be added to the metrics. That will allow us to build relations between metrics and/or add environment information.

### Describe the solution you'd like
We need to have the possibility to add some custom Tags which would be added to all metrics. 

Example of declaration:
```

      "My_Datadog_Metrics_Consumer": {
        "class": "Telemetry_Consumer",
        "type": "DataDog",
        "apiKey": "<redacted>",
        "dataType": "metrics",
        "customTags": [
          {
            "deploymentName": "f5aws01",
            "environment": "production",
            "instanceId": "i-0cd111111b224831"
          }
        ],
        "enable": true,
        "trace": false,
        "allowSelfSignedCert": false
      }

```

**Comment by dstokesf5** [comment]
Have you tried using the Data Modification feature to add tags at the system poller level? https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/data-modification.html#tag-property

**Comment by amolari** [comment]
@dstokesf5 
I've tried this declaration, but without success => I cannot see the tags added in the consumer trace output:

```
    "My_Metrics_Namespace": {
        "class": "Telemetry_Namespace",
        "My_Poller": {
            "class": "Telemetry_System_Poller",
            "interval": 60,
            "actions": [
             {
                 "enable": true,
                 "setTag": {
                     "deploymentInfo": {
                         "tag1": "tag1value",
                         "tag2": "Another tag"
                     }
                 }
             }
             ]
        },
[...]
```

What am I missing? As far as I understand, excludeData, IncludeData and IfAllMatched are optional (I would not want them anyway, as it should be applied to all keys in the SystemPoller)

**Comment by dstokesf5** [comment]
After further discussion with the developers I have learned that this method is not appropriate for adding global metric tags. I have added  AUTOTOOL-2795 to our internal product backlog to support specifying global tags on the DataDog consumer.

**Comment by amolari** [comment]
I guess it had been fixed in v1.25.0?

**Comment by dstokesf5** [comment]
This has been resolved in the 1.25.0 release.
